content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,207 @@
+repository.
+
+This means that the workflow will run whenever we run model registry actions,
+but we also want it to limit to specific ones for our specific workflow. That's
+where our GTO GitHub action comes into play - in the "parse" job of our workflow
+it parses all tags and if they are GTO tags, it gives us the name of the model,
+its version, stage (if any) and the event in the model registry.
+
+This is captured in the "parse" job which you can simply copy and paste into
+most CICD jobs of your own.
+
+```yaml
+# This job parses the git tag with the GTO GitHub Action to identify model registry actions
+parse:
+  runs-on: ubuntu-latest
+  steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: 'Parse GTO tag'
+      id: gto
+      uses: iterative/gto-action@v2
+  outputs:
+    event: ${{ steps.gto.outputs.event }}
+    name: ${{ steps.gto.outputs.name }}
+    stage: ${{ steps.gto.outputs.stage }}
+    version: ${{ steps.gto.outputs.version }}
+```
+
+<admon type="tip">
+
+If you cannot or don't want to use the GTO GitHub Action you can also use GTO
+directly with the [gto check-ref](/doc/gto/command-reference/check-ref) command.
+
+</admon>
+
+The next job called "deploy-model" actually performs the action. First, it uses
+the outputs of the parse job and checks whether the action should be performed.
+If the tag was produced by the model registry and if the corresponding action
+was assignment to the "prod" stage, it proceeds with the rest of the workflow.
+
+```yaml
+deploy-model:
+  needs: parse
+  if:
+    "${{ needs.parse.outputs.event == 'assignment' }} && ${{
+    needs.parse.outputs.stage == 'prod' }}"
+```
+
+The next step of the workflow sets up
+DVC (using a GitHub Action, but this can also be done manually, for example with
+pip).
+
+This allows us to run `dvc artifacts get` in the last step of the workflow to
+download the correct version of the model which can then be deployed or
+otherwise used in our CICD.
+
+```yaml
+- name: Get Model For Deployment
+  run: |
+    dvc artifacts get ${{ github.repositoryUrl }} ${{ needs.parse.outputs.name }} --rev ${{ needs.parse.outputs.version }}
+```
+
+Here, we are using the outputs of the `parse` job to specify the correct model
+version. Finally,
+[`github.repositoryUrl`](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)
+is a default environmental variable in GitHub which contains the URL of the
+current repository.
+
+You can use the following template to create your own Model Registry CICD
+actions on GitHub!
+
+```yaml
+name: Template CICD Action for DVC Model Registry
+
+on:
+  # the workflow is triggered whenever a tag is pushed to the repository
+  push:
+    tags:
+      - "*"
+
+jobs:
+
+  # This job parses the git tag with the GTO GitHub Action to identify model registry actions
+  parse:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
This might (or might not) be deleted, or partially deleted (just the fetch-depth) based on how we resolve the checkout issues.